followthemoney/proxy.py

```diff
@@ -276,7 +292,7 @@ def __eq__(self, other):
         return self.id == other.id
 
     @classmethod
-    def from_dict(cls, model, data, cleaned=True):
+    def from_dict(cls, model: Model, data, cleaned: bool=True):
```

Comment:
data: Union[Mapping, EntityProxy]

Reply:
Same, we can do that if `EntityProxy` inherits  `Mapping`